community/git.md

```diff
@@ -0,0 +1,19 @@
+# Sharing mods with git
+
+Author(s): code3z (@code3 on the forum)
+
+Git is a tool often used by developers to share and collaborate on great programs and docs (including this guide!). It allows developers to make individual changes to the main set of files and merge changes together.
+
+By using it to share your mods, you can make it easier to download a copy of all your mods, upload multiple files, put a license on all your mods, and allow others to suggest changes to make your mods better. If the Git provider has features like Issues and Projects, it can make it easier to collect bugs and plan new mods. You'll need an account on a git server provider such as Github (closed-source, free, most popular) or Gitlab (open-source, free, also popular). This guide won't teach you all about using git, but you can learn that using these guides:
```

Comment:
Pls don’t mix up git, GitHub & a regular folder/archive.

- license mods – even with a single mod file
- d/l a copy of all your mods, upload multiple files, get suggestions – folder/archive
- git is not a service, so no git provider

Reply:
> git is not a service, so no git provider

Sorry, I don’t understand. Aren’t there many providers of git servers?

Reply:
I can’t argue about git *server* providers, but a git provider is a nonsense (unless you are talking about your distro’s packager or whoever, & even then I’d be careful).

I’d choose a git (repository) hosting platform instead.